enhancements/storage/csi-driver-install.md

```diff
@@ -144,7 +182,8 @@ Bases on current upstream plans & assumptions.
 
 ### OCP-4.5 (Kubernetes 1.18)
 * We support installation of AWS EBS (tech preview) and Manila CSI (GA) drivers with in-tree drivers. User can use both
-  drivers at the same time. The default storage class is for in-tree volume plugin (unless cluster admin changes it).
+  in-tree volume plugin and CSI drivers at the same time. The default storage class is for in-tree volume plugin (unless
+  cluster admin changes it).
```

Comment:
nit: there's no in-tree plugin for Manila

Does CSO refrain itself from setting the default SC back to the intree one?

Reply:
CSO already does not change the default storage class if admin has changed it to something custom. We don't plan any change there.